### DISCUSSION TITLE
Is a GUI available for the CLI version on Windows 10?

### DISCUSSION THREAD

**Discussion by Waxwood** [yatima1460/Drill#79] [Q&A]
Hi, Thanks for your excellent work on DRILL.
I use it on my home Linux system but would like to also use it on Windows. 

Is a GUI available for the CLI version on Windows 10?
Thank you!

**Comment by yatima1460** [comment]
Hello!!

Thanks for using Drill!

Sadly I am pretty stupid when it comes to GUIs, I barely had the skill to make the Linux one, but if someone wants to contribute with one for Windows it would be nice;

also support for D is not that great...

i was actually thinking to rewrite it into another language 🤔

**Comment by yatima1460** [comment]
well i converted it to dotnet7

so a WinForms UI will come!
